app/templates/support/index.html

```diff
@@ -0,0 +1,52 @@
+{% extends "_base_page.html" %}
+{% block page_title %}What are you trying to do? - Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with
+    items = [
+      {
+          "link": url_for('main.index'),
+          "label": "Digital Marketplace"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+<div class="support-page">
+  
+
+  <div class="grid">
+    
+      <header class="page-heading">
+        <h1>
+          What are you trying to do?
+        </h1>
+      </header>
+  
+      {% with smaller = true, heading = "Sell services" %} {% include "toolkit/page-heading.html" %} {% endwith %}
```

Comment:
This will create multiple `<h1>`s on the page, which we don't really want (it's bad for accessibility - we generally only have one top-level heading per page).  These should be `<h2>`s.  Kat recently added some styles for these in `dmspeak` (copied from GOV.UK styles - see here: https://github.com/alphagov/digitalmarketplace-frontend-toolkit/commit/200483ce04fe3715d0a624aedbdd0f33b90f084d), which might work here?